tasks/main.yml

```diff
@@ -1,14 +1,21 @@
 ---
+- name: Gather OS specific variables.
+  include_vars: "{{ item }}"
+  with_first_found:
+      - "vars/{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
+      - "vars/{{ ansible_distribution }}.yml"
+      - "defaults/main.yml"
+
 - name: Ensure Pip is installed.
   package:
-    name: "{{ pip_package }}"
-    state: present
+      name: "{{ pip_package }}"
```

Comment:
Can you keep the same spacing (2 space indentation) instead of changing the spacing for all these values? It makes the PR harder to review with all the spacing changes.